hazelcast/src/test/resources/hazelcast-client-multicast-plugin-non-default-port.xml

```diff
@@ -27,6 +27,9 @@
     <network>
         <discovery-strategies>
             <discovery-strategy enabled="true" class="com.hazelcast.spi.discovery.multicast.MulticastDiscoveryStrategy">
+                <properties>
+                    <property name="port">54329</property>
```

Comment:
How is this port number determined? Is it just one of the non-used ones in the codebase?

Reply:
Exactly like that

Reply:
In the Go client, we changed all code that uses port numbers to automatically generate them to fix some flaky tests. That may also be useful in this codebase, but of course not in this PR.